pkg/security/tests/schemas/exit.schema.json

```diff
@@ -10,29 +18,19 @@
                     "required" : [
                         "cause",
                         "code"
-                    ]
-                },
-                "process": {
-                    "type": "object",
-                    "required": [
-                        "pid",
-                        "ppid",
-                        "tid",
-                        "uid",
-                        "gid",
-                        "user",
-                        "group",
-                        "comm",
-                        "tty",
-                        "fork_time",
-                        "exec_time",
-                        "exit_time"
-                    ]
+                    ],
```

Comment:
We don't have a process context anymore ?

Reply:
You're right, I'll add it back while keeping the process attributes not required.

Reply:
I think we could use the process_context.json file